src/tools/create-app-request-client.js

```diff
@@ -43,7 +44,15 @@ const createAppRequestClient = (input, options) => {
   const httpAfters = [
     prepareResponse,
     logResponse
-  ].concat(ensureArray(app.afterResponse));
+  ];
+
+  if (app.authentication) {
+    if (app.authentication.type === 'oauth2' && _.get(app, 'authentication.oauth2Config.autoRefresh', false)) {
```

Comment:
You don't need to pass `fasle` to `_.get` as missing values return `undefined` by default.